W3ChampionsStatisticService/Matches/MatchRepository.cs

```diff
@@ -13,8 +13,11 @@ namespace W3ChampionsStatisticService.Matches
 {
     public class MatchRepository : MongoDbRepositoryBase, IMatchRepository
     {
+        private readonly List<OnGoingMatchup> _onGoingMatchups;
```

Comment:
You don't need to assign this? Just use OnGoingMatchesMemoryStore.Matches

Reply:
fixed